akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcClusterSpec.scala

```diff
@@ -88,7 +88,7 @@ abstract class MultiDcSpec
         testConductor.blackhole(first, third, Direction.Both).await
       }
       runOn(first, second, third, fourth) {
-        awaitAssert(clusterView.unreachableMembers should not be empty)
+        awaitAssert(clusterView.reachability.allUnreachable should not be empty)
       }
```

Comment:
To add some more integration style of test for this we can add a `cluster.subscribe` to a TestProbe here and see that we get the expected events. Well here it shouldn't get any `UnreachableMember` (which is also a good test) but it should get `UnreachableDataCenter` when @2m has finished that.